OfficeApplication/OfficeApplication/Administration.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System;
+
+namespace OfficeApplication
+{
+    public class Administration : Department, IEmployeeManagementService
+    {
+        public EmployeeEntities Context = new EmployeeEntities();
+        public void AddEmployee(TblEmployee employee)
+        {
+            Context.TblEmployees.Add(employee);
+            Context.SaveChanges();
+        }
+
+        public void RemoveEmployee(int removeId)
```

Comment:
Rename removeId by employeeId as RemoveEmployee shows it's removing.
same for other operations.